app/src/main/java/com/kuit/kuit5/ui/asset/component/AssetConfirmItem.kt

```diff
@@ -0,0 +1,79 @@
+package com.kuit.kuit5.ui.asset.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.kuit.kuit5.R
+import com.kuit.kuit5.model.AssetData
+
+@Composable
+fun AssetConfirmItem(
+    modifier: Modifier = Modifier,
+    confirm:AssetData
+) {
+    Row(
+        modifier=modifier
+            .fillMaxWidth(),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween
+    ){
+        Row(
+            verticalAlignment = Alignment.CenterVertically
+        ){
+            Icon(
+                modifier = modifier
+                    .size(30.dp),
+                painter = painterResource(id = confirm.icon),
+                contentDescription = "asset icon",
+                tint = Color.Unspecified
+            )
+            Spacer(modifier.size(10.dp))
+            Text(
+                text = confirm.name,
+                fontSize = 14.sp
+            )
+        }
+        Column(){
+            Text(
+                text=confirm.value,
+                color=Color(0xFF9FA580),
+                fontSize = 14.sp
+            )
+            HorizontalDivider(
+                modifier=modifier
+                    .width(50.dp),
+                color= Color(0xFFD9D9D9)
+            )
```

Comment:
```suggestion
        Column(
            modifier = Modifier
            .width(
                intrinsicSize = IntrinsicSize.Min
            )
        ){
            Text(
                text = confirm.value,
                color = Color(0xFF9FA580),
                fontSize = 14.sp
            )
            HorizontalDivider(
                modifier = Modifier
                    .fillMaxWidth(),
                color = Color(0xFFD9D9D9)
            )
        }
```
로 선언하면 텍스트의 크게 맞게 Divider 의 길이를 정할 수 있어요.
IntrinsicSize.min 의 의미는 하위 요소들 중 가장 작은 친구의 크기로 설정하겠다는 뜻입니다.
그래서 Text => 14.sp , Divider -> MaxSize 이기 때문에 더 작은 14.sp 에 해당하는 width 가 설정된겁니다.